reference/shipping_provider.yml

```diff
@@ -1974,9 +1907,7 @@ components:
               items:
                 type: object
                 title: Customer Group
-                description: The Group (if any) that this customer is in. This will be
-                  default to zero if the customer is not in a group or is a
-                  guest.
+                description: The Group (if any) that this customer is in. This will be default to zero if the customer is not in a group or is a guest.
```

Comment:
```suggestion
                description: The Group (if any) that this customer is in. This will default to zero if the customer is not in a group or is a guest.
```